logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1

```diff
@@ -15,35 +15,31 @@
 #>
 param(
     [Parameter(
-        Mandatory = $true,
-        HelpMessage = "Path to the sqlpackage.exe is required.`n`rExample: C:/Program Files/Microsoft SQL Server/150/DAC/bin"
+        HelpMessage = 'Path to the sqlpackage.exe is required.`n`rExample: C:/Program Files/Microsoft SQL Server/150/DAC/bin'
     )]
     [ValidateNotNullOrEmpty()]
-    [string] $sqlPackagePath,
+    [string] $sqlPackagePath = 'C:/Program Files/Microsoft SQL Server/150/DAC/bin',
 
     [Parameter(
-        Mandatory = $true,
-        HelpMessage = "Path the the output folder is required.`n`rExample: C:/tmp/artifacts"
+        HelpMessage = 'Path the the output folder is required.`n`rExample: C:/tmp/EdFi.CloudOds'
     )]
     [ValidateNotNullOrEmpty()]
-    [string] $artifactPath,
+    [string] $artifactPath = (Join-Path $env:temp 'EdFi.CloudOds'),
 
+    [string] $packageName = 'EdFi.CloudODS',
+
+    [Obsolete("This parameter is deprecated, and will be removed in the near future.")]
     [switch] $WhatIf
 )
 
+if (-not (Test-Path $sqlPackagePath)) { throw "Could not find sqlpackage.exe at $sqlPackagePath" }
+
 $ErrorActionPreference = 'Stop'
 
-$repositoryNames = @('Ed-Fi-Ods', 'Ed-Fi-ODS-Implementation', 'Ed-Fi-ODS-Tools\Application\EdFi.Ods.AdminApp.Web')
-& "$PSScriptRoot/../../../../../logistics/scripts/modules/load-path-resolver.ps1" $repositoryNames
-$implementationRepo = Get-Item "$PSScriptRoot/../../../../.." | Select-Object -Expand Name
-$env:toolsPath = $toolsPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'tools')
+$repositoryNames = @('Ed-Fi-Ods', 'Ed-Fi-ODS-Implementation', 'Ed-Fi-ODS-AdminApp/Application/EdFi.Ods.AdminApp.Web')
```

Comment:
I've noticed a lot of switch back and forth between single quotes and double quotes. Is there any particular style defined as "standard"?

Reply:
double quotes allows for variable substitution. The C# equivalent would be wrapping every string with `$""` even when not using string interpolation. Powershell doesn't warn you when you are using double quotes and no variables, but I try to fix them where I can.